_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,428 @@
+
+You can run reports automatically by setting up a cron job. OpenSearch Reporting CLI supports any cron job use case, such as setting a specified time of day.
+
+### Prerequisites
+
+You need a machine with cron installed and root access privileges.
+
+### Step 1: Install the reporting CLI
+
+Install the Reporting CLI by running the following command:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+### Step 2: Specify the details for the report
+
+Open the crontab editor by running the following command:
+
+```
+crontab -e
+```
+In the crontab editor, enter the report request. For example, the following example shows a cron report that runs every day at 8am.
+
+```
+0 8 * * * opensearch-reporting-cli -u https://playground.opensearch.org/app/dashboards#/view/084aed50-6f48-11ed-a3d5-1ddbf0afc873 -e ses -s <sender_email> -r <recipient_email>
+```
+
+## Scheduling reports with AWS Lambda
+
+You can use AWS Lambda with the Reporting CLI tool to specify an AWS Lambda function to trigger the report generation.
+
+This requires that you use an AM64 system and Docker.
+
+### Prerequisites
+
+To use the Reporting CLI with AWS Lambda, you need to do the following preliminary steps.
+
+- Get an AWS account. For instructions, see [Creating an AWS account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-creating.html) in the AWS Account Management reference guide.
+- Get a Dockerfile to assemble an image. For Docker instructions, see [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
+- Set up an Amazon Elastic Container Registry (ECR). For instructions, see [Getting started with Amazon ECR using the AWS Management Console](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html).
+
+
+### Step 1: Create a container image with a Dockerfile
+
+You need to assemble the image by running a Dockerfile. When you run the Dockerfile, it downloads the OpenSearch artifact required to use the Reporting CLI.
+
+Copy the following sample configurations into a Dockerfile.
+
+```dockerfile
+
+# Define function directory
+ARG FUNCTION_DIR="/function"
+
+# Base image of the docker container
+FROM node:lts-slim as build-image
+
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+
+# AWS Lambda runtime dependencies
+RUN apt-get update && \
+    apt-get install -y \
+        g++ \
+        make \
+        unzip \
+        libcurl4-openssl-dev \
+        autoconf \
+        automake \
+        libtool \
+        cmake \
+        python3 \
+        libkrb5-dev \
+        curl
+
+# Copy function code
+WORKDIR ${FUNCTION_DIR}
+RUN curl -LJO https://artifacts.opensearch.org/reporting-cli/opensearch-reporting-cli-1.0.0.tgz
+RUN tar -xzf opensearch-reporting-cli-1.0.0.tgz
+RUN mv package/* .
+RUN npm install && npm install aws-lambda-ric
+
+# Build Stage 2: Copy Build Stage 1 files in to Stage 2. Install chromium dependencies and chromium.
```

Comment:
comment is inaccurate: it installs google-chrome, not chromium, then removes google-chrome to keep the dependencies. Maybe
```suggestion
# Build Stage 2: Copy Build Stage 1 files in to Stage 2. Install chrome dependencies.
```